zips/zip-0235.md

```diff
@@ -0,0 +1,152 @@
+network.
+
+“We” - the ZIP authors, owners listed in the above front matter
+
+# Abstract
+
+We propose to burn 60% of transaction fees, while the remaining 40% be directed
+as before, providing a deflationary effect, and building the groundwork for
+long-term support of the Zcash network via the new block subsidy rules proposed
+by ZIP-234.
+
+# Motivation
+
+ZIP-233 ("Network Sustainability Mechanism: Burning") describes a method by
+which ZEC can be burned to support network sustainability.
+
+By introducing a requirement that a certain proportion of transaction fees be
+burned, we ensure that ZEC will be removed from circulating supply to contribute
+to the long-term sustainability of the network as described below:
+
+## Benefits to the Network
+
+1. **Network Sustainability**: This mechanism involves temporarily reducing the
+   supply of ZEC, similar to asset burning in Ethereum’s EIP-1559, but with
+   long-term sustainability benefits, as the burned funds effectively boost
+   future mining rewards, making it an attractive option for current and future
+   Zcash users.
+2. **Incentivizing Transaction Inclusion**: By maintaining a 40% share of
+   transaction fees for miners, we continue incentivizing miners to prioritize
+   including transactions in their blocks. This helps ensure the continued
+   efficient processing of transactions and supports a robust and responsive
+   network.
+3. **Aligning Interests:** Burning transaction fees aligns the interests
+   of miners with the long-term health of the network, incentivizing them
+   to prioritize security and efficiency. As miners focus on maintaining a
+   robust network, overall adoption and usage can increase, leading to higher
+   transaction volumes in the long run. This structure discourages short-term
+   profit-seeking behaviors, as miners benefit more from a stable and thriving
+   ecosystem than from engaging in malicious activities that could undermine the
+   network's reputation and security.
+4. **Future-Proofing the Network**: Burning a portion of transaction fees
+   is a forward-looking approach that prepares the Zcash network for future
+   challenges and opportunities. It establishes a financial buffer that can be
+   instrumental in addressing unforeseen issues and seizing strategic advantages
+   as the Zcash ecosystem evolves.
+
+# Requirements
+
+* For each block, at least 60% (rounded down) of the total fees are to be
+burned.
+* No restrictions are placed on the destination of the remaining proportion of
+fees.
+* Any fractions are rounded in favor of the miner.
+
+# Specification
+
+## Consensus Rule Changes
+
+For a given block, the coinbase transaction MUST have a `burnAmount` that is
+greater than or equal to `floor(transactionFees * 6 / 10)`.
+
+In transaction versions without the `burnAmount` field, the total output value
+of the coinbase transaction MUST be the value of the block subsidy, plus
+`ceil(transactionFees * 4 / 10)`.
```

Comment:
This no longer works given [ZIP 236](https://zips.z.cash/zip-0236), which will activate with NU6. ZIP 236 requires blocks to balance exactly, and therefore requires coinbase transactions to claim the maximum amount available. The simplest way to fix it would be to require coinbase transactions to be a new version that has the `burnAmount` field.

Reply:
> ZIP 236 requires [...] coinbase transactions to claim the maximum amount available.

The intention here is to redefine "maximum amount available". I think we can do this and still require that everything balances exactly, right?

Reply:
I've [modified this paragraph](https://github.com/zcash/zips/compare/92fa29b11bbc89c1c0e458f0b063c2dc925b979b..e43a8afb3f705f7424825fb15f9004f2c81102f9) to read:

> Previous transaction versions are not supported for coinbase transactions, due to there being no explicit mechanism to burn the required funds.